web-app/js/portal/cart/PythonDownloadHandler.js

```diff
@@ -37,7 +37,7 @@ Portal.cart.PythonDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
         var _this = this;
 
         return function(collection) {
-            return collection.getLayerState().getSelectedLayer().getFeatureRequestUrl(
+            return OpenLayers.Layer.WMS.prototype.getFeatureRequestUrl(
```

Comment:
I'm not sure about this myself - i.e. where should this method live?  It's pretty WMS/WFS specific, so tacking it on to `OpenLayers.Layer.WMS`, as it already was, seems ok.  But the way it's called here, and elsewhere, is a bit unusual in our code base.